code/modules/mob/living/carbon/human/species/_species.dm

```diff
@@ -461,6 +461,9 @@
 		var/obj/item/organ/external/temp = M.bodyparts_by_name["r_hand"]
 		if(M.hand)
 			temp = M.bodyparts_by_name["l_hand"]
+		if(temp && temp.disabled)
```

Comment:
A neat bit of 512 that simplifies these kinds of checks. It also really minorly improves performance but not enough to matter

```suggestion
		if(temp?.disabled)
```